repro_first_trial.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+        name: containervm-deployment
+spec:
+        replicas: 11
```

Comment:
the default namespace has a limitrange that sets cpu=100m which you can work around by setting your own limits of cpu=10m, that'd given you more pods per node and (probably) a faster repro